Tools/python/Sample.py

```diff
@@ -9,7 +9,7 @@
 
 xrootd_sites = {
     'T3_US_FNALLPC': 'root://cmsxrootd.fnal.gov/', #'root://cmseos.fnal.gov/',
-    'T1_US_FNAL_Disk': 'root://cmsxrootd-site.fnal.gov/',
+    'T1_US_FNAL_Disk': 'root://cmsxrootd.fnal.gov/', #'root://cmsxrootd.fnal.gov/', #'root://cmsxrootd-site.fnal.gov/',
```

Comment:
If a sample is at FNAL, the -site version is better, because it will find the file directly rather than going through the redirector (which can be slower). Maybe someone found that they had to do this to get things working though...?